### PR TITLE
refactor: implement "chart" component

### PR DESCRIPTION
**Pull request by leMaur**
## Summary
https://app.clickup.com/t/pvbv7a

~~Waiting for https://github.com/ArkEcosystem/laravel-ui/pull/592~~

This PRs implements the new reusable Chart component.

### How to test
1. open `/statistics` page
1. expect to see charts without issues

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Review by ItsANameToo — APPROVED** [review]

**Comment by ItsANameToo** [comment]
will merge this once it's merged on UI

**Review by alexbarnsley — APPROVED** [review]

**Pull request merged**
